### ISSUE TITLE
[Filters] (minor) Skip filters link visible?

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-evaluation-tools-list#42]
The skip filters link does not become visible when tabbed to:

```<a href="#tools-list" class="button button--skip-link">Skip filters</a>```

This could be confusing for sighted keyboard users as there is no focus visibility.
